### PR TITLE
WIP: make oss build work with preview4

### PR DESCRIPTION
**Pull request by KevinRansom**

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -299,23 +299,36 @@ call src\update.cmd signonly
...
 if exist "%ProgramFiles(x86)%\Microsoft Visual Studio 12.0\common7\ide\devenv.exe" set VisualStudioVersion=12.0
 if exist "%ProgramFiles%\Microsoft Visual Studio 12.0\common7\ide\devenv.exe" set VisualStudioVersion=12.0
 
 :vsversionset
 if '%VisualStudioVersion%' == '' echo Error: Could not find an installation of Visual Studio && goto :failure
 
-if exist "%ProgramFiles(x86)%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe" set _msbuildexe="%ProgramFiles(x86)%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"
-if exist "%ProgramFiles%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"      set _msbuildexe="%ProgramFiles%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"
-if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe. && goto :failure
+if exist "%VS150COMNTOOLS%..\..\MSBuild\15.0\Bin\MSBuild.exe" (
```
Comment (otawfik-ms) [inline build.cmd]: MSBuild is now applocal. Good find. :+1:

File: vsintegration/src/FSharp.ProjectSystem.FSharp/ProjectSystem.fsproj [hunk 2]
```diff
@@ -163,7 +164,8 @@
     <Win32Resource>$(IntermediateOutputPath)\ProjectResources.rc.res</Win32Resource>
   </PropertyGroup>
   <Target Name="BeforeBuild">
-    <Exec Command="&quot;$(ProgramFiles)\Windows Kits\8.1\bin\x86\rc.exe&quot; /fo $(IntermediateOutputPath)$(RCResourceFile).res $(RCResourceFile)" />
+    <Exec Command="&quot;$(ProgramFiles)\Windows Kits\8.1\bin\x86\rc.exe&quot; /fo $(IntermediateOutputPath)$(RCResourceFile).res $(RCResourceFile)" Condition="Exists('$(ProgramFiles)\Windows Kits\8.1\bin\x86\rc.exe')" />
+    <Exec Command="&quot;$(ProgramFiles)\Windows Kits\10\bin\x86\rc.exe&quot; /fo $(IntermediateOutputPath)$(RCResourceFile).res $(RCResourceFile)" Condition="Exists('$(ProgramFiles)\Windows Kits\10\bin\x86\rc.exe')" />
```
Comment (otawfik-ms) [inline vsintegration/src/FSharp.ProjectSystem.FSharp/ProjectSystem.fsproj]: &quot; ?

**Comment by otawfik-ms** [comment]
LGTM :+1:

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Pull request merged**